docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,112 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Improve performance of queries with grouping on joins. ({issue}`23812`)
+* TBD Spooling protocol is stable. Link to docs! ({issue}`24214`)
+* TBD Improve performance of querying `system.jdbc.tables` for multiple catalogs. ({issue}`24159`)
+* Support partial update. ({issue}`24075`)
+
+## Security
+
+* Add support [data access control with Apache
```

Comment:
@kokosing fyi

Reply:
"Add support **for** ..."?

column-masking -> column masking
row-filtering -> row filtering